RecoEgamma/ElectronIdentification/test/pyfwlite_test.py

```diff
@@ -0,0 +1,80 @@
+#! /usr/bin/env python
```

Comment:
Better move this to a common area.
This will avoid confusion when development in muons will require edits in RecoEgamma/ElectronIdentification code